test/functional/repository/find-options/repository-find-options.ts

```diff
@@ -256,9 +253,9 @@ describe("repository > find options > comment", () => {
     beforeEach(() => reloadTestingDatabases(connections))
     after(async () => {
         await closeTestingConnections(connections)
-        if (fs.existsSync(logPath)) {
-            await promisify(unlink)(logPath)
-        }
+        try {
+            fs.unlink(logPath)
```

Comment:
await

Reply:
This was a good catch, I somehow didn't see it. 

There's an ESLint rule for this: `@typescript-eslint/no-floating-promises`, and it's in the `recommended-type-checked` config. We should consider enabling these rules.

Reply:
absolutely